charts/kafka-operator/templates/operator-deployment-with-webhook.yaml

```diff
@@ -125,6 +125,10 @@ spec:
       {{- if .Values.additionalVolumes }}
       {{- include "chart.additionalVolumes" . | nindent 8 }}
       {{- end }}
+      {{- if .Values.securityContext }}
+      securityContext:
```

Comment:
`spec.containers[*] `    also has own property `securityContext`

e.g. 

https://github.com/banzaicloud/logging-operator/blob/master/charts/logging-operator/templates/deployment.yaml#L48-L52

Reply:
You're right. I'm going to introduce a separate field for pod and container level `securityContext`.